_inc/lib/icalendar-reader.php

```diff
@@ -96,7 +96,7 @@ function apply_timezone_offset( $events ) {
 		} else {
 			// If the timezone isn't set then the GMT offset must be set.
 			// generate a DateInterval object from the timezone offset
-			$gmt_offset = get_option( 'gmt_offset' ) * HOUR_IN_MINUTES;
+			$gmt_offset = get_option( 'gmt_offset' ) * HOUR_IN_SECONDS;
 			$timezone_offset_interval = date_interval_create_from_date_string( "{$gmt_offset} minutes" );
```

Comment:
That's a good point, nice catch! Updated in ccc4fb978e65b731d644fca1041df831b36eeeee